composer.json

```diff
@@ -58,15 +58,16 @@
             "Composer\\Config::disableProcessTimeout",
             "@cs",
             "@deduplicate",
-            "@inspect",
```

Comment:
I typically run Deptrac first because it is so much faster than PHPStan; if there is a violation it ends faster. I don't have a strong preference though.

Reply:
Done.

Reply:
By the way, the most time consuming command is `rector` in `analyze`.
Actually `test` is much faster than `analyze` in Shield.

Reply:
It's true for this library but this comes from my template I use across many libraries where that isn't always the case. I'm fine for this to deviate, just explaining where this comes from!